api/src/test/resources/org/openmrs/validator/include/ObsValidatorTest.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<dataset>
+ 
+  <person person_id="7" gender="F" birthdate="1976-08-25 00:00:00.0" birthdate_estimated="1" dead="false" creator="1" date_created="2006-01-18 00:00:00.0" voided="false" uuid="5946f880-b197-400b-9caa-a3c661d23041"/>
+  <person person_id="8" gender="F" birthdate="1976-08-26 00:00:00.0" birthdate_estimated="1" dead="false" creator="1" date_created="2007-01-18 00:00:00.0" voided="false" uuid="5946f880-b197-400b-9caa-a3c661d23045"/>
+    
+  <encounter encounter_id="3" encounter_type="2" patient_id="7" location_id="1" form_id="1" encounter_datetime="2008-08-01 00:00:00.0" creator="1" date_created="2008-08-18 14:09:05.0" voided="false" uuid="6519d653-393b-4118-9c83-a3715b82d4ac"/>
+  
+  <obs obs_id="7" person_id="7" status="FINAL" concept_id="5089" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="50.0" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="[NULL]" value_datetime="[NULL]" value_drug="[NULL]" uuid="39fb7f47-e80a-4056-9285-bd798be13c63"/>
+ 
```

Comment:
you have unnecessary empty lines here

Reply:
@mozzy11  okie let me remove them

Reply:
removed the unwanted spaces on this @mozzy11

Reply:
@dkayiwa  does this look good enough to be merged?

Reply:
@HerbertYiga i can here the obs person id = 7 = encounter patient id.

clarify in your test cases what your exactly testing